<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaintainerClass</class>
 <widget class="QMainWindow" name="MaintainerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>259</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Maintainer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButtonSettingDelete">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>151</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>削除設定</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSettingBackup">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>151</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>バックアップ設定</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSetting">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>151</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>ログ設定</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>361</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelAction">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Action</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPath">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Path</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonExecuteNow">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>151</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>今実行</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
